src/core/Clock.js

```diff
@@ -14,9 +18,9 @@ function Clock( autoStart ) {
 
 }
 
-Object.assign( Clock.prototype, {
+Clock.prototype = {
 
-	start: function () {
+	start() {
```

Comment:
The Rollup build's `buble` step currently converts things like `start() {` to `start: function start() {`

Bonus: the output now has named functions. whereas prior to this PR the output has only `start: function() {`. This makes debugging easier. :tada: